docs/integrations/sources/asana.md

```diff
@@ -64,6 +75,26 @@ The connector is restricted by normal Asana [requests limitation](https://develo
 | `array`                  | `array`      |
 | `object`                 | `object`     |
 
+## Limitations & Troubleshooting
+
+<details>
+<summary>
+Expand to see details about Asana connector limitations and troubleshooting.
+</summary>
+
+### Connector limitations
+
+#### Rate limiting
+
+The connector is restricted by normal Asana [requests limitation](https://developers.asana.com/docs/rate-limits).
```

Comment:
The connector is restricted by [Asana rate limits](https://developers.asana.com/docs/rate-limits).